src/components/ui/autocomplete.svelte

```diff
@@ -34,7 +34,7 @@
   let optionGen;
   let promises = [];
   let hasMoreOptions;
-  let selected = new Set(values.map(opt => opt.name));
```

Comment:
Nah, I don't approve of those changes. This is a UI component, the data format shouldn't have to imply that it is only used for users with a full name and the extra data is always an email. That's why in the Create Project I've simply rewired the objects coming from the backend before passing them over to the Autocomplete component.

Reply:
So do you suggest to change the function in "@/constants/autocomplete-options.js" or the wiring between importing the function and passing it to the autocomplete component?